README.md

```diff
@@ -244,8 +244,8 @@ Since a `<Flex />` component works the same way as a DOM one, you can easily mak
 
 ### Measuring the container
 
-When building responsive layouts, you might need to syncronize the size of the 3D Flex container with the DOM, for example, to syncronize scroll position or the height of a scroll container.
-To make it easier, you can use the `onReflow` callback on the root `<Flex>` component, that will be called every time the flex layout is recalculated - e.g. when any content changes.
+When building responsive layouts you might need to syncronize the size of the 3D Flex container with the DOM, for example to synchronize scroll position or the modify the height of a scroll container.
```

Comment:
`or the modify the height`
is this right?

Reply:
yeah that was something I'm not sure about - both don't read easily

syncing the scroll position is obvious - what are we talking about when we mention height of a scroll container?

syncing the height of a scroll container to the scroll position..?

Reply:
I mean, isn't supposed to say _or **TO** modify_?

Reply:
hehe, yes that works